usaspending_api/settings.py

```diff
@@ -72,6 +72,8 @@
     "-nonprod" if DOWNLOAD_ENV != "production" else ""
 )
 IDV_DOWNLOAD_README_FILE_PATH = os.path.join(BASE_DIR, "usaspending_api/data/idv_download_readme.txt")
+ASSISTANCE_DOWNLOAD_README_FILE_PATH = os.path.join(BASE_DIR, "usaspending_api/data/AssistanceSummary_download_readme .txt")
```

Comment:
Doesn't really matter since it matches the file, but you have an extra space after `readme` in the file name: `AssistanceSummary_download_readme .txt` instead of `AssistanceSummary_download_readme.txt`.